bin/varnishd/cache/cache_deliver_proc.c

```diff
@@ -137,8 +161,10 @@ VDP_close(struct req *req)
 /*--------------------------------------------------------------------*/
 
 static int v_matchproto_(objiterate_f)
-vdp_objiterator(void *priv, int flush, const void *ptr, ssize_t len)
+vdp_objiterator(void *priv, int flush, int last, const void *ptr, ssize_t len)
```

Comment:
From the object iterator POV, `flush` means the content is ephemereal, or that the pointers are only valid up until the function returns. `last` is the very last call. So they are not really the same, though for the consumer (VDP), `last` would also imply `flush` as we would then be finished delivering the response and a flush is executed anyways.

So yes, this could be combined to a bit field, or even an enum like in the VDP interface with last implying flush. But since the two args are not really the same, and this isn't a very critical code path (only called once per delivery), I think it's fine to use two unsigned args.